boards/nxp/imx95_evk/doc/index.rst

```diff
@@ -0,0 +1,121 @@
+.. _imx95_evk:
+
+NXP i.MX95 EVK (Cortex-A55)
+############################
+
+Overview
+********
+
+i.MX95 MPU is composed of one cluster of 6x Cortex®-A55 cores and a single
+Cortex®-M7 core and a Cortex®-M33 core. Zephyr OS is ported to run on one of
+the Cortex®-A55 core.
+
+- Board features:
+
+  - RAM: 16GB LPDDR5
+  - Storage:
+
+    - Micron 64GB eMMC
+    - microSD Socket
+  - Wireless:
+
+    - One Wi-Fi/Bluetooth Murata Type-2EL module based on NXP AW693 chip
+      supporting 2x2 Wi-Fi 6 and Bluetooth 5.2
+  - USB:
+
+    - Two USB ports: Type A (USB 2.0) and Type C (USB 3.0)
+  - Ethernet
+  - PCI-E:
+
+    - M.2
+    - x4 slot
+  - LEDs:
+
+    - 4x Power status LED
+    - 3x UART LED
+  - Debug
+
+    - JTAG 20-pin connector
+    - MicroUSB for UART debug, three COM ports for A55, M7 and M33
+
+
+Supported Features
+==================
+
+The Zephyr imx95_evk board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| GIC-v4    | on-chip    | interrupt controller                |
++-----------+------------+-------------------------------------+
+| ARM TIMER | on-chip    | system clock                        |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+
```

Comment:
The above table is for A-core target, add a new table to list the hardware features used by M-core.

Reply:
Okay.